### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by dzg** [dhleong/pepper-mint#13]
Is this project being updated?

I don't think Mint uses `ius_session` or `thx_guid` any more.

**Comment by dhleong** [comment]
You wanna be a bit more specific there? 

This project works well enough on https://github.com/dhleong/mint-spendable but it depends on what you're trying to use it for. I do get an error every now and again when logging in, but usually when I retry right after everything works. If it consistently fails usually the Mint website is being wacky and a few hours later everything's fine again.

I'd be happy to work with you on a PR but last I checked the original python API also doesn't work super consistently these days, especially with Mint adding Google's "transparent captcha."

**Comment by dzg** [comment]
Sorry for my vagueness ... I came looking for an alternative mroony's mintapi which stopped working well since Mint changed something on their end (see https://github.com/mrooney/mintapi/issues/129).

Your docs say to grab `ius_session` and `thx_guid` from a Mint session ... but AFAIK those aren't being used anymore.

**Comment by dhleong** [comment]
Have you tried the version without passing those?

```javascript
require('pepper-mint')(username, password)
.then(function(mint) {
    // NOTE: this is spelled out to clarify the format
    // of the sessionCookies property
    persistCookies({
        ius_session: mint.sessionCookies.ius_session,
        thx_guid: mint.sessionCookies.thx_guid,
    });
});
```

That's the flow I'm using. It should open a Chrome instance and login for you, then you can save the cookies and pass them back on subsequent login attempts.

**Comment by dhleong** [comment]
The current version did seem to be broken for a bit, so I worked on [this branch](https://github.com/dhleong/pepper-mint/tree/dhleong/update-auth). Feel free to try it out and see if it fixes things for you. For what it's worth, though, I just tried out the old version and it seems to be working again....

**Comment by dhleong** [comment]
@dzg Not sure if you're still interested or if you ever tried the branch, but I finally got around to ironing things out and have merged the new auth flow into master and released v2.0.0. I'm going to close this issue since it's been months since I've heard from you, but feel free to open a new ticket if you encounter new problems with the new version of PepperMint.
